scripts/status_docker

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# List of pids if the chain is running
+# No output if the chain isn't running
+
+echo -n "all running nodes: "
+echo
+
+docker ps | grep '\d\-Node'
```

Comment:
you can use the filter:
docker ps -f network=docker-network_vpcbr 
To get running nodes

Reply:
as mentioned in another pr, my network name is different, not sure why